src/sider/aktivitetskrav/vurdering/SendForhandsvarselSkjema.tsx

```diff
@@ -39,7 +44,16 @@ const brevMalTexts: {
   [Brevmal.UTLAND]: "Bosatt i utlandet",
 };
 
-const defaultValues = { begrunnelse: "", arsak: undefined };
+export interface SendForhandsvarselSkjemaValues
+  extends AktivitetskravSkjemaValues {
+  fristDato: Date;
+}
```

Comment:
Trenger denne å importeres andre steder? (`export`) 😊 Synes også personlig at å extende `AktivitetskravSkjemaValues` som egentlig bare har en property (`begrunnelse`) blir mer forvirrende enn tydelig. Foreslår å bare ha begge propertiene i dette interfacet 😊👇
```suggestion
interface SendForhandsvarselSkjemaValues
  begrunnelse: string;
  fristDato: Date;
}
```

Reply:
Synes extensions av interfaces skal være ganske omstendelig før det bidrar til bedre leselighet, men det er jo bare min subjektive mening 😊 På et eller annet nivå mener jeg det blir mer jobb å legge til rette for [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), enn å bare skrive det flere ganger.

Hvis du tenker å extende for å legge på `begrunnelse` er greit så kan vi godt bare gjøre det nå 🙏